ParisBNF/abb/BNFabb20.xml

```diff
@@ -33,12 +33,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     </msIdentifier>
                     <msContents> 
                         <msItem xml:id="ms_i1">
+                            <locus from="3r" to="127v"/>
+                        <msItem xml:id="ms_i1.1">
                             <locus from="3r" to="125v"/>
                             <title type="complete" ref="LIT3171Commen"/>
```

Comment:
as specified in https://github.com/BetaMasaheft/Documentation/issues/2311, LIT3171Commen#Vita is a chapter inside of LIT3171Commen (which is msItem 1). 
here you can specify the relevant div (s) or insert a descriptive title, as, 
for the chapters in LIT3171Commen please check the record.

the structure is (if we follow the proposal of Dorothea @thea-m )

msItem 1 LIT3171Commen 
  msItem 1.1 commentary
  msItem 1.2 biography by Giyorgis Walda Amid


OR (if we follow what Mersha suggested in the issue)

msItem 1 Full dossier containing the commentary and the biography (then we need a new CAe ID here)
 msItem 1.1. Commentary LIT3171Commen 
 msItem 1.2 Biography by Giyorgis (then we need a new CAe ID here too, and edit other records referring to LIT3171Commen#Vita)